cli/src/helpers.rs

```diff
@@ -93,17 +98,27 @@ pub fn command_create_solido(
     config: &mut SnapshotConfig,
     opts: &CreateSolidoOpts,
 ) -> Result<CreateSolidoOutput> {
-    let lido_keypair = Keypair::new();
+    let lido_signer = {
+        if opts.solido_key_path() != &PathBuf::default() {
+            // If we've been given a solido private key, use it to create the solido instance.
+            let signer = get_signer(opts.solido_key_path().clone());
+            signer
```

Comment:
This can be just `get_signer` without binding it to a variable, right?

Reply:
Yes, also the following one. Changed it.